x-pack/plugins/ml/public/explorer/explorer_charts/explorer_chart_single_metric.test.js

```diff
@@ -128,7 +128,7 @@ describe('ExplorerChart', () => {
     expect(paths[0].getAttribute('class')).toBe('domain');
     expect(paths[1].getAttribute('class')).toBe('domain');
     expect(paths[2].getAttribute('class')).toBe('values-line');
-    expect(paths[2].getAttribute('d')).toBe('MNaN,159.33024504444444MNaN,9.166257955555556LNaN,169.60736875555557');
+    expect(paths[2].getAttribute('d')).toBe('MNaN,159.33024504444444ZMNaN,9.166257955555556LNaN,169.60736875555557');
```

Comment:
@elastic/ml-ui Does this look alright with you?

Reply:
@spalger yes this edit to one of the ML d3 chart tests LGTM